### DISCUSSION TITLE
show chapters in itunes and on iphone

### DISCUSSION THREAD

**Discussion by BeckeBauer** [yermak/AudioBookConverter#181] [General]
Not sure if I am able to handle the program correctly. When I create an audiobook from several mp3, itunes shows no chapters (when I open the created m4b file with foobar2000, chapters are shown) - any suggestions how to config so that itunes shows chapters?

**Comment by yermak** [comment]
Hi BeckeBauer,

Let's break this discussion in two parts: iphone and itunes.

Iphone: 
I just retested chapters are shown and it's possible to navigate using them. (top righ corner of the screen when book is playing)

Itunes:
Chapters are **only shown** when you click on small window where book name is displayed - click there switch display information.
![image](https://user-images.githubusercontent.com/380491/102063085-7dad8780-3ded-11eb-87a6-a055e1697518.png)
Click!
![image](https://user-images.githubusercontent.com/380491/102062939-4c34bc00-3ded-11eb-8463-495583116fbc.png)
Unfortunately, i have not found a way how to navigate between chapters in itunes (ha-ha, they say apple products are intuitive) 

If you really want to see chapters in itunes, then you need select **split by chapters** when creating a book, then you will have multiple m4b files in a folder and instead of this in authors view of itunes
![image](https://user-images.githubusercontent.com/380491/102063365-d8df7a00-3ded-11eb-8210-6fc2a55d77e7.png)
You will see book like this:
![image](https://user-images.githubusercontent.com/380491/102063846-76d34480-3dee-11eb-8e87-89b428b695e8.png)

But, honestly, i'm not sure why would you want to go this route as you could do exactly same with mp3 in itunes.

My summary: itunes is useless as player for audiobooks - its only purpose to transfer audiobooks to iphone. But there is a plan to eliminate even this dependency - stay tuned ;)

**Comment by BeckeBauer** [comment]
Thanks for the feedback. I will check it out on the iPhone (so far, I tried only iTunes assuming that if it works there, it will work on the phone not expecting that it might work on the phone but not in iTunes)

One more thing: is there a way to configure that the chapters keep the titles instead of getting the name of the audio book with the extension chapter 1, chapter 2 and so on?

**Reply by BeckeBauer** [reply]
I checked it out on my iphone (iPhone4 with latest compatible iOS 7.1) but unfortunately, chapters are not shown (I am also unable to install foobar2000 due to the old iOS)... I assume that it is simply to old and I will have to invest in a new phone in order to enjoy the luxury of chapters 😞

**Reply by yermak** [reply]
> Thanks for the feedback. I will check it out on the iPhone (so far, I tried only iTunes assuming that if it works there, it will work on the phone not expecting that it might work on the phone but not in iTunes)
> 
> One more thing: is there a way to configure that the chapters keep the titles instead of getting the name of the audio book with the extension chapter 1, chapter 2 and so on?

I assume you mean to keep chapters' file names when you select - split by chapters... Unfortunately, this is not currently possible - all chapters inherit book name and "chapter N" as suffix.  However, it is possible to configure meta information about chapter - most likely you player will display it.

**Reply by yermak** [reply]
> I checked it out on my iphone (iPhone4 with latest compatible iOS 7.1) but unfortunately, chapters are not shown (I am also unable to install foobar2000 due to the old iOS)... I assume that it is simply to old and I will have to invest in a new phone in order to enjoy the luxury of chapters 😞

Sorry, I comment on such model and iOS version.

**Reply by BeckeBauer** [reply]
> 
> 
> > One more thing: is there a way to configure that the chapters keep the titles instead of getting the name of the audio book with the extension chapter 1, chapter 2 and so on?
> 
> I assume you mean to keep chapters' file names when you select - split by chapters... Unfortunately, this is not currently possible - all chapters inherit book name and "chapter N" as suffix. However, it is possible to configure meta information about chapter - most likely you player will display it.

I meant to have the name of the mp3 file as chapter title (probably stored as meta information) after creation of the audiobook.m4b from several mp3 files. 
Example: The audiobook is named MyBook and the first mp3 file is named "the example", the second mp3 file is named "to be continued" and so on. After creation of the audiobook MyBook.m4b, the first chapter is named "MyBook Chapter 1" instead of "the example", the second chapter is "MyBook Chapter 2" instead of "to be continued" and so on. Not sure if you mean by "it is possible to configure meta information about chapter" that I can achieve it since I assume that the chapter title are meta information? If so, how?

**Reply by yermak** [reply]
Hi,
Please select chapter in the tree and click edit button, you will see few options there from using filename or any tag to manually configuring each chapter.

> Thanks for the feedback. I will check it out on the iPhone (so far, I tried only iTunes assuming that if it works there, it will work on the phone not expecting that it might work on the phone but not in iTunes)
> 
> One more thing: is there a way to configure that the chapters keep the titles instead of getting the name of the audio book with the extension chapter 1, chapter 2 and so on?
